### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by katcipis**

### CODE REVIEW

**Comment by katcipis** [comment]
Tests passes here....travis is timeouting =/. Can anyone also run to confirm that it is working ?

File: Makefile [hunk 1]
```diff
@@ -60,20 +64,20 @@ cpu=10
...
+test-integration: imagedev
 	./hack/run.sh $(gotest) -timeout $(integration_timeout) -run=$(run) ./... $(gotestargs)
 
-test-examples: image
+test-examples: imagedev
 	./hack/run.sh $(gotest) -timeout $(examples_timeout) -tags=examples -run=TestExamples $(gotestargs)
 
 # It is recommended to use this locally. It takes too much time for the CI
-test-all: test
+test-all: testdev
```
Comment (vitorarins) [inline Makefile]: There is no `testdev` target. Did you mean `imagedev`?

Comment (katcipis) [reply Makefile]: This target seems ancient =P...never use it

Comment (katcipis) [reply Makefile]: Remembered and fixed it =P

**Pull request merged**